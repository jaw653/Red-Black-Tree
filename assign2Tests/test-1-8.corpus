ijrh
xzxz
nswo
gmwv
jkiu
wgpy
xxno
sfda
qujn
ugmj
ykxh
wveh
hocg
uret
qthi
ykko
htdb
bqmc
aljz
gpgp
dkxx
bbtu
wacu
npkw
kqzm
glqi
wbje
qqtt
arse
ulyr
ocnd
razc
qyqy
jggf
hsjx
iesi
xkos
ymlm
rypi
yomq
ocox
kxet
ppsx
vkht
xynv
kyhb
yymz
oypc
debn
dhjs
wbrt
oaml
yzgl
xnoy
naxb
bmfg
sgvv
pepn
gigu
jtfj
unuu
alsp
npqo
bxuw
fqru
wjhe
rnya
ifmd
sizu
trjh
gbxj
auff
mzbi
iinz
ynci
vqln
ymku
dvbl
zaxz
uchg
dirl
tent
trbo
hpcf
bozf
jcqk
dnkz
stix
bzlw
fypy
rrny
gpdh
ffoq
jhbm
wnoo
gwoj
vzga
zvyq
olou
cteh
asyk
zbwy
omou
icfd
blfc
jesx
rguw
cyfc
sdmu
glsu
xgqg
jwlm
jrps
xjqq
skmu
ksyc
xlwe
yoax
xrfi
pruy
klrj
