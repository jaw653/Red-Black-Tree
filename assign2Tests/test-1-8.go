i ijrh
i xzxz
i nswo
i gmwv
i jkiu
i wgpy
i xxno
i sfda
i qujn
i ugmj
i ykxh
i wveh
i hocg
i uret
i qthi
i ykko
i htdb
i bqmc
i aljz
i gpgp
i dkxx
i bbtu
i wacu
i npkw
i kqzm
i glqi
i wbje
i qqtt
i arse
i ulyr
i ocnd
i razc
i qyqy
i jggf
i hsjx
i iesi
i xkos
i ymlm
i rypi
i yomq
i ocox
i kxet
i ppsx
i vkht
i xynv
i kyhb
i yymz
i oypc
i debn
i dhjs
i wbrt
i oaml
i yzgl
i xnoy
i naxb
i bmfg
i sgvv
i pepn
i gigu
i jtfj
i unuu
i alsp
i npqo
i bxuw
i fqru
i wjhe
i rnya
i ifmd
i sizu
i trjh
i gbxj
i auff
i mzbi
i iinz
i ynci
i vqln
i ymku
i dvbl
i zaxz
i uchg
i dirl
i tent
i trbo
i hpcf
i bozf
i jcqk
i dnkz
i stix
i bzlw
i fypy
i rrny
i gpdh
i ffoq
i jhbm
i wnoo
i gwoj
i vzga
i zvyq
i olou
i cteh
i asyk
i zbwy
i omou
i icfd
i blfc
i jesx
i rguw
i cyfc
i sdmu
i glsu
i xgqg
i jwlm
i jrps
i xjqq
i skmu
i ksyc
i xlwe
i yoax
i xrfi
i pruy
i klrj
s
r
