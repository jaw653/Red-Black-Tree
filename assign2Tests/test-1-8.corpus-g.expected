0: ijrh-2(ijrh-2)-
1: gmwv-2(ijrh-2)-l xzxz-2(ijrh-2)-r
2: bqmc-2(gmwv-2)-l hocg-2(gmwv-2)-r nswo-2(xzxz-2)-l ykxh-2(xzxz-2)-r
3: aljz-2(bqmc-2)-l dkxx-2(bqmc-2)-r gpgp-2(hocg-2)-l htdb-2(hocg-2)-r jkiu-2(nswo-2)-l wgpy-2(nswo-2)-r =ykko-2(ykxh-2)-l ymlm-2(ykxh-2)-r
4: bbtu-2(aljz-2)-r debn-2(dkxx-2)-l glqi-2(dkxx-2)-r =gpdh-2(gpgp-2)-l =gwoj-2(gpgp-2)-r hsjx-2(htdb-2)-l iesi-2(htdb-2)-r jggf-2(jkiu-2)-l npkw-2(jkiu-2)-r sfda-2(wgpy-2)-l xxno-2(wgpy-2)-r =ymku-2(ymlm-2)-l yomq-2(ymlm-2)-r
5: arse-2(bbtu-2)-l bmfg-2(bbtu-2)-r bxuw-2(debn-2)-l dhjs-2(debn-2)-r gigu-2(glqi-2)-l =glsu-2(glqi-2)-r =hpcf-2(hsjx-2)-l =icfd-2(iesi-2)-l ifmd-2(iesi-2)-r jcqk-2(jggf-2)-l =jhbm-2(jggf-2)-r kqzm-2(npkw-2)-l =npqo-2(npkw-2)-r qujn-2(sfda-2)-l ugmj-2(sfda-2)-r wveh-2(xxno-2)-l =xynv-2(xxno-2)-r ynci-2(yomq-2)-l yymz-2(yomq-2)-r
6: =alsp-2(arse-2)-l auff-2(arse-2)-r =blfc-2(bmfg-2)-l =bozf-2(bmfg-2)-r bzlw-2(bxuw-2)-r =dirl-2(dhjs-2)-r fqru-2(gigu-2)-l =iinz-2(ifmd-2)-r =jesx-2(jcqk-2)-r jtfj-2(kqzm-2)-l kxet-2(kqzm-2)-r qthi-2(qujn-2)-l razc-2(qujn-2)-r sgvv-2(ugmj-2)-l uret-2(ugmj-2)-r wjhe-2(wveh-2)-l xkos-2(wveh-2)-r =yoax-2(ynci-2)-r yzgl-2(yymz-2)-r
7: =asyk-2(auff-2)-l cteh-2(bzlw-2)-r dvbl-2(fqru-2)-l gbxj-2(fqru-2)-r =jrps-2(jtfj-2)-l jwlm-2(jtfj-2)-r =ksyc-2(kxet-2)-l kyhb-2(kxet-2)-r qqtt-2(qthi-2)-l =qyqy-2(razc-2)-l rypi-2(razc-2)-r sizu-2(sgvv-2)-r ulyr-2(uret-2)-l wacu-2(uret-2)-r =wnoo-2(wjhe-2)-r xgqg-2(xkos-2)-l xnoy-2(xkos-2)-r zaxz-2(yzgl-2)-r
8: =cyfc-2(cteh-2)-r =dnkz-2(dvbl-2)-l =ffoq-2(dvbl-2)-r =fypy-2(gbxj-2)-l =klrj-2(jwlm-2)-r naxb-2(kyhb-2)-r ocnd-2(qqtt-2)-l rnya-2(rypi-2)-l =sdmu-2(rypi-2)-r trjh-2(sizu-2)-r =unuu-2(ulyr-2)-r vkht-2(wacu-2)-l wbje-2(wacu-2)-r =xjqq-2(xgqg-2)-r =xlwe-2(xnoy-2)-l =xrfi-2(xnoy-2)-r zvyq-2(zaxz-2)-r
9: =mzbi-2(naxb-2)-l =oaml-2(ocnd-2)-l ocox-2(ocnd-2)-r =rguw-2(rnya-2)-l =rrny-2(rnya-2)-r tent-2(trjh-2)-l =uchg-2(trjh-2)-r vqln-2(vkht-2)-r =wbrt-2(wbje-2)-r =zbwy-2(zvyq-2)-l
10: ppsx-2(ocox-2)-r stix-2(tent-2)-l =trbo-2(tent-2)-r =vzga-2(vqln-2)-r
11: oypc-2(ppsx-2)-l =pruy-2(ppsx-2)-r =skmu-2(stix-2)-l
12: olou-2(oypc-2)-l =pepn-2(oypc-2)-r
13: =omou-2(olou-2)-r
Words/Phrases: 242
Nodes: 121
Minimum depth: 4
Maximum depth: 14
